rest_framework/views.py

```diff
@@ -524,7 +525,7 @@ async def async_dispatch(self, request, *args, **kwargs):
         self.headers = self.default_response_headers  # deprecate?
 
         try:
-            self.initial(request, *args, **kwargs)
+            sync_to_async(self.initial)(request, *args, **kwargs)
```

Comment:
Shouldn't this statement have an `await`?

Reply:
You are right, I have pushed the fix.

Reply:
@em1208 I am concerned the tests did not hit this. Can you add a test to cover this?

Reply:
@em1208 I'm concerned the tests did not hit this. Could you add a test to cover this please?